### PR TITLE
Fixed ChatView backgroundColor customization

### PR DESCRIPTION
**Pull request by EgorovEI**
Fixed calling super setup method in ChatView.
Set clear tableView backgroundColor.

MOB-1209

### CODE REVIEW

File: GliaWidgets/View/Call/CallView.swift [hunk 1]
```diff
@@ -81,6 +81,54 @@ class CallView: EngagementView {
         adjustLocalVideoFrameAfterLayout()
     }
 
+    override func setup() {
```
Comment (kuldar-daniel) [inline GliaWidgets/View/Call/CallView.swift:84]: super.setup() missing here. Is this intentional?

**Review by dukhovnyi — APPROVED** [review]
Looks good 👍

File: GliaWidgets/View/Call/CallView.swift (hunk 1 above)
Comment (EgorovEI) [reply GliaWidgets/View/Call/CallView.swift:84]: It's ok. As described in task "Right now, almost everything is configurable, but the background for the middle section of the chat (the white portion) is not exposed", we need to fix only backgroundColor in chat

**Review by igorkravchenko — APPROVED** [review]
Looks good to me 👍

**Review by kuldar-daniel — APPROVED** [review]

**Pull request merged**